src/Backend/OnboardingInsuranceAPI/Areas/User/PostUser.cs

```diff
@@ -22,11 +22,13 @@ public async Task CreateUser(string pid, string email)
         var user = await _context.Users.FirstOrDefaultAsync(x => x.Pid == pid);//check if user exist
```

Comment:
As we discussed before the handler should be called something like RegisterUser and then there should be method Handle.

Reply:
Ah, yeah You are right, I remember but missed this becuase didnt pay attention on old code